contrib/python/pypodman/pypodman/lib/actions/_create_args.py

```diff
@@ -0,0 +1,394 @@
+            '--label',
+            '-l',
+            help=('Add metadata to a container'
+                  ' (e.g., --label com.example.key=value)'))
+        parser.add_argument(
+            '--label-file', help='Read in a line delimited file of labels')
+        parser.add_argument(
+            '--log-driver',
+            choices=['json-file', 'journald'],
+            help='Logging driver for the container.')
+        parser.add_argument(
+            '--log-opt',
+            action='append',
+            help='Logging driver specific options')
+        parser.add_argument(
+            '--memory', '-m', action=UnitAction, help='Memory limit')
+        parser.add_argument(
+            '--memory-reservation',
+            action=UnitAction,
+            help='Memory soft limit')
+        parser.add_argument(
+            '--memory-swap',
+            action=UnitAction,
+            help=('A limit value equal to memory plus swap.'
+                  'Must be used with the --memory flag'))
+        parser.add_argument(
+            '--memory-swappiness',
+            choices=range(0, 100),
+            metavar='[0-100]',
+            help="Tune a container's memory swappiness behavior")
+        parser.add_argument('--name', help='Assign a name to the container')
+        parser.add_argument(
+            '--network',
+            metavar='BRIDGE',
+            help=('Set the Network mode for the container.'))
+        parser.add_argument(
+            '--oom-kill-disable',
+            action=BooleanAction,
+            help='Whether to disable OOM Killer for the container or not')
+        parser.add_argument(
+            '--oom-score-adj',
+            choices=range(-1000, 1000),
+            metavar='[-1000-1000]',
+            help="Tune the host's OOM preferences for containers")
+        parser.add_argument('--pid', help='Set the PID mode for the container')
+        parser.add_argument(
+            '--pids-limit',
+            type=int,
+            metavar='LIMIT',
+            help=("Tune the container's pids limit."
+                  " Set -1 to have unlimited pids for the container."))
+        parser.add_argument('--pod', help='Run container in an existing pod')
+        parser.add_argument(
+            '--privileged',
+            action=BooleanAction,
+            help='Give extended privileges to this container.')
+        parser.add_argument(
+            '--publish',
+            '-p',
+            metavar='RANGE',
+            help="Publish a container's port, or range of ports, to the host")
+        parser.add_argument(
+            '--publish-all',
+            '-P',
+            action=BooleanAction,
+            help='Publish all exposed ports to random'
+            ' ports on the host interfaces'
+            '(Default: False)')
+        parser.add_argument(
+            '--quiet',
+            '-q',
+            action='store_true',
+            help='Suppress output information when pulling images')
```

Comment:
Does varlink support sending this information?

Reply:
Currently always "quiet".  varlink allows progress updates, but that implies the go code is modified to capture the output and proxy it to the client.